pkg/apis/core/validation/validation.go

```diff
@@ -3511,6 +3511,8 @@ type PodValidationOptions struct {
 	AllowWindowsHostProcessField bool
 	// Allow more DNSSearchPaths and longer DNSSearchListChars
 	AllowExpandedDNSConfig bool
+	// Allow invalid topologySpreadConstraint labelSelector for backward compatibility
```

Comment:
I am not sure this will break anything, since it will rejected at the runtime anyway, https://github.com/kubernetes/kubernetes/pull/111441/files#r957044565